lxd/device/nic_bridged.go

```diff
@@ -309,6 +310,14 @@ func (d *nicBridged) Start() (*deviceConfig.RunConfig, error) {
 		return nil, err
 	}
 
+	// Attempt to enable port isolation
+	if !strings.HasPrefix(saveData["host_name"], "tunnel.") && shared.IsTrue(d.config["security.port_isolation"]) {
```

Comment:
@matthewa150 it would be good to have a comment here as why the logic to for `!strings.HasPrefix(saveData["host_name"], "tunnel.")` is here?

Also I don't think this logic is correct. From what I can see there is no other reference to "tunnel." keys in the NIC bridged driver, so it will never be populated.

I believe @stgraber meant in his comment on https://github.com/lxc/lxd/issues/7949#issue-710480600 "There should be an exception for external interfaces and for tunnel devices when the system itself isn't a router (so where the tunnel will be used as an uplink)." - that isolation mode should be left disabled on the bridge ports that connect to tunnels or external interfaces. 

So for NIC devices these will always connect to an instance and won't be tunnel/external ports.

I believe just removing the ` !strings.HasPrefix(saveData["host_name"], "tunnel.") &&` bit entirely will suffice.

This will prevent the instances communicating with each other, but will still allow them to communicate with the LXD host and any tunnel or external bridge ports.

Reply:
Ah, that's a misunderstanding on my end. I removed the expression in question.